streams/ca.m3u

```diff
@@ -68,7 +68,7 @@ https://cbcrclinear-tor.akamaized.net/hls/live/2042768/CBCRCLINEAR_TOR_14/master
 #EXTINF:-1 tvg-id="CBETDT.ca",CBC Windsor (CBET-DT) (720p) [Geo-blocked]
 https://cbcrclinear-tor.akamaized.net/hls/live/2042759/CBCRCLINEAR_TOR_5/master5.m3u8
 #EXTINF:-1 tvg-id="CHBLTV.ca",CHBL-TV
-https://fl3.moveonjoy.com/CA_GLOBAL/index.m3u8
+https://fl4.moveonjoy.com/CA_GLOBAL/index.m3u8

```

Comment:
Please move every link from country playlists into us_moveonjoy if you have time. Thank you!